openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,53 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(TimeStampedModel):
+    """
+    Model to save invitations, sent out to users for becoming admin of business unit(s).
+    """
+
+    name = models.CharField(verbose_name=_('Name'), max_length=255, null=True, blank=True, )
+    email = models.EmailField(verbose_name=_('Email'), max_length=255, unique=True, null=False, blank=False, )
```

Comment:
Please remove the `null=False` and blank=False` attributes. They are not needed since by default the null and blank attributes are set to False.